docs/docs/modeling.md

```diff
@@ -114,6 +114,34 @@ classDiagram
 
 ## AssignmentSheet - 分担票
 
+Tenant's task assignment table.  
+The assignmentSheet is including startDate,endDate and TenantsWork.
+
+Constraints：
+
+- The AssignmentSheet cannot exist without StartDate, EndDate and TenantsWork.
+
+### startDate
+
+The start date of the task to be performed by Tenant.
+
+Constraints：
+
+- Owner must select startDate.
+- The time zone is PST (Pacific Standard Time).
+
+### endDate:
+
+The end day the task must be completed.
```

Comment:
Thanks for the work, in my opinion, "The deadline for completing the task." is easier to grasp